paddle/fluid/operators/cinn_launch_op_test.cc

```diff
@@ -170,7 +170,133 @@ TEST(CinnLaunchOpTest, TestElementwiseAddPass) {
+}
+
+TEST(CinnLaunchOpHelperTest, TestGetCinnTensorsFromCompiledScope) {
+  CinnScope cinn_scope;
+  cinn_scope.Var<CinnTensor>("cinn_var1");
+  cinn_scope.Var<CinnTensor>("cinn_var2");
+  cinn_scope.Var<CinnTensor>("cinn_var3");
+
+  auto cinn_tensors =
+      GetCinnTensorsFromCompiledScope({"cinn_var1", "cinn_var3"}, cinn_scope);
+  ASSERT_EQ(cinn_tensors.size(), 2);
+  ASSERT_EQ(cinn_tensors.front().get(),
+            cinn_scope.GetTensor("cinn_var1").get());
+  ASSERT_EQ(cinn_tensors.back().get(), cinn_scope.GetTensor("cinn_var3").get());
+  ASSERT_THROW(
+      GetCinnTensorsFromCompiledScope({"cinn_var1", "not_exist"}, cinn_scope),
+      paddle::platform::EnforceNotMet);
+}
+
+TEST(CinnLaunchOpHelperTest, TestCheckTensorEquivalent) {
+  platform::CPUPlace place;
+  framework::Scope scope;
+  auto* tensor1 = scope.Var("var1")->GetMutable<LoDTensor>();
+  tensor1->mutable_data<float>(framework::make_ddim({5, 8}), place);
+
+  CinnScope cinn_scope;
+  cinn_scope.Var<CinnTensor>("cinn_var1");
+  auto cinn_tensor1 = cinn_scope.GetTensor("cinn_var1");
+  cinn_tensor1->Resize(CinnShape({5, 8}));
+
+  ASSERT_NO_THROW(CheckTensorEquivalent("var1", tensor1, cinn_tensor1));
+  cinn_tensor1->Resize(CinnShape({5, 7}));
+  ASSERT_THROW(CheckTensorEquivalent("var1", tensor1, cinn_tensor1),
+               paddle::platform::EnforceNotMet);
+  auto tensor2 = scope.Var("var2")->GetMutable<LoDTensor>();
+  ASSERT_THROW(CheckTensorEquivalent("var2", tensor2, cinn_tensor1),
+               paddle::platform::EnforceNotMet);
+}
+
+TEST(CinnLaunchOpHelperTest, TestTensorMutableDataWithCinnInfo) {
+  platform::CPUPlace place;
+  framework::Scope scope;
+  auto* tensor1 = scope.Var("var1")->GetMutable<LoDTensor>();
+  CinnScope cinn_scope;
+  cinn_scope.Var<CinnTensor>("cinn_var1");
+  auto cinn_tensor1 = cinn_scope.GetTensor("cinn_var1");
+  cinn_tensor1->Resize(CinnShape({5, 8}));
+
+  ASSERT_NO_THROW(TensorMutableDataWithCinnInfo(place, cinn_tensor1, tensor1));
+  ASSERT_TRUE(tensor1->IsInitialized());
+  ASSERT_EQ(tensor1->dims(), framework::make_ddim({5, 8}));
+}
+
+TEST(CinnLaunchOpHelperTest, TestSeperateTempVar) {
+  CinnScope cinn_scope;
+  cinn_scope.Var<CinnTensor>("cinn_var1");
+  cinn_scope.Var<CinnTensor>("cinn_var2");
+  cinn_scope.Var<CinnTensor>("cinn_var3");
+  cinn_scope.Var<CinnTensor>("cinn_var4");
+
+  auto temp_names =
+      SeperateTempVar(cinn_scope, {"cinn_var1", "cinn_var2"}, {"cinn_var4"});
+  ASSERT_EQ(temp_names.size(), 1);
+  EXPECT_EQ(temp_names.front(), "cinn_var3");
+  ASSERT_THROW(
+      SeperateTempVar(cinn_scope, {"cinn_var1", "not_exist"}, {"cinn_var4"}),
+      paddle::platform::EnforceNotMet);
+}
+
+TEST(CinnLaunchOpHelperTest, TestShareTensorWithCinnBuffer) {
+  platform::CPUPlace place;
+  framework::Scope scope;
+  auto* tensor1 = scope.Var("var1")->GetMutable<LoDTensor>();
+  tensor1->mutable_data<float>(framework::make_ddim({5, 6}), place);
+  auto* data1 = tensor1->data<float>();
+  data1[0] = 9.99;
```

Comment:
建议：`float`字面值都加`f`后缀
```suggestion
  data1[0] = 9.99f;
```

Reply:
fixed